internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -100,6 +104,11 @@ func (opts *Opts) Run() error {
 		return err
 	}
 
+	runMongoShell, err := opts.askMongoShellQuestion()
+	if err != nil {
+		return err
+	}
```

Comment:
[q/nit] asking to launch the cluster before it's created may be too soon, asking about what version of the shell to use is ok at this point but I don't think we should ask to launch or download until we are done creating (this is personal preference not a blocking thing)
Also for context, we should tell users that's safe to `ctrl+c` when "creating the cluster", so if you don't want to wait you don't have too, reason why I don't think we should ask about shell so soon

Reply:
I will add a comment in the spec to discuss where we should ask these questions

Reply:
added